cli/endpoints/online/profiling/profiling_job_tmpl.yml

```diff
@@ -0,0 +1,25 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/latest/commandJob.schema.json
+command: >
+  profiling.sh -p {inputs.payload}
+experiment_name: profiling-job
+environment:
+  docker:
+    image: docker.io/rachyong/profilers:latest
+environment_variables:
+  ONLINE_ENDPOINT: "<% ENDPOINT_NAME %>"
+  DEPLOYMENT: "<% DEPLOYMENT_NAME %>"
+  PROFILING_TOOL: "<% PROFILING_TOOL %>"
+  DURATION: "<% DURATION %>"
+  CONNECTIONS: "<% CONNECTIONS %>"
+  TARGET_RPS: "<% TARGET_RPS %>"
+  CLIENTS: "<% CLIENTS %>"
+  TIMEOUT: "<% TIMEOUT %>"
+  THREAD: "<% THREAD %>"
+compute:
+  target: <% COMPUTE_NAME %>
```

Comment:
syntax should be: azureml:compute_name

Reply:
added "azureml" part to the yaml template